src/main/java/carbonconfiglib/gui/widgets/GuiUtils.java

```diff
@@ -46,7 +43,7 @@ public class GuiUtils
 	private static final ScissorsStack STACK = new ScissorsStack();
 
 	public static FormattedText ellipsizeStyled(Component text, int maxWidth, Font font) {
```

Comment:
Technically these functions can be removed since they are no longer used thanks to the scrolling text actually being backportable :)